### DISCUSSION TITLE
Should passlib still be a part of the security tutorial?

### DISCUSSION THREAD

**Discussion by johannaschwarz** [fastapi/fastapi#11437] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from passlib.context import CryptContext

pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")

def verify_password(plain_password, hashed_password):
    return pwd_context.verify(plain_password, hashed_password)
```

### Description

In the [security tutorial](https://fastapi.tiangolo.com/tutorial/security/oauth2-jwt/#install-passlib) passlib is used for password hashing. Using it in this way results in an error (see [this issue ](https://foss.heptapod.net/python-libs/passlib/-/issues/190) in passlib). 

For the sake of this tutorial it should be enough to just use bcrypt and avoid this error. 

### Operating System

macOS

### FastAPI Version

0.110.1

### Pydantic Version

2.6.3

### Python Version

3.11.8

**Comment by YuriiMotov** [comment]
Duplicate of: https://github.com/fastapi/fastapi/discussions/11773
